actors/evm/src/lib.rs

```diff
@@ -273,6 +301,15 @@ pub struct ConstructorParams {
     pub initcode: RawBytes,
 }
 
+#[derive(Serialize_tuple, Deserialize_tuple)]
+pub struct DelegateCallParams {
+    pub code: Cid,
+    /// The contract invocation parameters
+    pub input: RawBytes,
```

Comment:
Is the input CBOR? If not, this should be `Vec<u8>` (with a `serde(with...)` decorator to encode it correctly).

Reply:
Its just bytes.

Reply:
fixed.